### PR TITLE
Update ssdlite-mbv3 modelzoo

### PR DESCRIPTION
**Pull request by still-wait**
1. update mAP.
2. add fpn.
3. add quant model.

### CODE REVIEW

File: configs/mobile/README.md [hunk 1]
```diff
@@ -37,7 +40,9 @@ PaddleDetection目前提供一系列针对移动应用进行优化的模型，
 |                  | SD625   | SD835   | SD845   | SD855   | Kirin 970 | Kirin 980 |
 |------------------|---------|---------|---------|---------|-----------|-----------|
 | SSDLite Large    | 289.071 | 134.408 | 91.933  | 48.2206 | 144.914   | 55.1186   |
+| SSDLite Large Quant |  |  |   |  |    |    |
```
Comment (willthefrog) [inline configs/mobile/README.md:43]: missing numbers?

File: configs/mobile/README.md [hunk 2]
```diff
@@ -48,16 +53,28 @@ PaddleDetection目前提供一系列针对移动应用进行优化的模型，
...
+python slim/quantization/train.py --not_quant_pattern res_conv1 se_block \
+        -c configs/ssd/ssdlite_mobilenet_v3_large.yml \
+        --eval
+```
+更多量化教程请参考[模型量化压缩教程](../../docs/advanced_tutorials/slim/quantization/QUANTIZATION.md)
 
 ## YOLOv3剪裁说明
 
-首先对YOLO检测头进行剪裁，然后再使用 YOLOv3-ResNet34 作为teacher网络对剪裁后的模型进行蒸馏, teacher网络在COCO的mAP为31.4 (输入大小320\*320).
+首先对YOLO检测头进行剪裁，然后再使用 YOLOv3-ResNet34 作为teacher网络对剪裁后的模型进行蒸馏, teacher网络在COCO上的mAP为31.4 (输入大小320\*320).
```
Comment (willthefrog) [inline configs/mobile/README.md:77]: what happened to this line?

Comment (still-wait) [reply configs/mobile/README.md:77]: `在COCO的mAP为`-->`在COCO上的mAP为`

File: configs/mobile/README.md (hunk 1 above)
Comment (still-wait) [reply configs/mobile/README.md:43]: 数据待加入，QA提测，需要先合入master

File: configs/mobile/README_en.md [hunk 3]
```diff
@@ -48,13 +53,27 @@ This directory contains models optimized for mobile applications, at present the
...
+    | SSDLite Small Quant |  |  |  | |   |    |
     | YOLOv3 baseline  | 413.486 | 184.248 | 133.624 | 75.7354 | 202.263   | 126.435   |
     | YOLOv3 prune     | 98.5472 | 53.6228 | 34.4306 | 21.3112 | 44.0722   | 31.201    |
     | Cascade RCNN 320 | 131.515 | 59.6026 | 39.4338 | 23.5802 | 58.5046   | 36.9486   |
     | Cascade RCNN 640 | 473.083 | 224.543 | 156.205 | 100.686 | 231.108   | 138.391   |
 
 
+## Notes on SSDLite quantization
+
+In the SSDLite model, we use a complete quantitative training method to train the model. A total of 400,000 rounds are trained under the 8-card GPU. During the training, `res_conv1` and `se_block` are fixed and not trained. The execution instructions are:
```
Comment (willthefrog) [inline configs/mobile/README_en.md:67]: We use a complete quantitative training method to train the SSDLite model. It is trained for a total of 400,000 rounds with the 8-card GPU. We freeze `res_conv1` and `se_block` . The command used is listed bellow:

Comment (still-wait) [reply configs/mobile/README_en.md]: Done，thx~

File: configs/mobile/README.md (hunk 2 above)
Comment (willthefrog) [reply configs/mobile/README.md:77]: okay, I see

**Pull request merged**